src/test/java/org/cactoos/io/InputStreamOfTest.java

```diff
@@ -103,7 +103,7 @@ public void readsFileContent() throws Exception {
         final String content = "Content in a file";
         new LengthOf(
             new TeeInput(content, file)
-        ).intValue();
+        ).value();
         new Assertion<>(
```

Comment:
@baudoliver7 good eye! apparently @vzurauskas and @fanifieiev are to be blame for the PR #1323 and bad review :) and also @paulodamaso for accepting it as ARC :laughing:

Reply:
@victornoel 😆

Reply:
@victornoel Ha, I always forget these affirmations somewhere. :)

Reply:
@vzurauskas hehe, yep, that could almost be argument in favour of procedural code for writing tests ^^

Or would need to change the test framework so that the test methods return the `Assertion` and the framework is responsible of calling `affirm`! :stuck_out_tongue: